composer.json

```diff
@@ -12,12 +12,12 @@
 	],
 	"require": {
 		"php": "~7.2",
-		"symfony/config": "~4.0",
-		"symfony/console": "~4.0",
-		"symfony/dependency-injection": "~4.0",
-		"symfony/http-kernel": "~4.0",
-		"symfony/twig-bundle": "~4.0",
-		"symfony/yaml": "~4.0",
+		"symfony/config": "^4.4|^5.0",
+		"symfony/console": "4.4|^5.0",
```

Comment:
shouldn't it be `^4.4`?

Reply:
Sure. I'm just spamming force pushes here. Did not want to bother with old symfony versions, had no idea someone might see it. Thanks!